platform/ios/src/MGLScaleBarView.h

```diff
@@ -0,0 +1,5 @@
+#import <UIKit/UIKit.h>
+
+@interface MGLScaleBarView : UIView
+
```

Comment:
Should we expose any sort of styling properties publicly? Colors, typeface, position?